src/chocolatey.resources/helpers/functions/Install-ChocolateyPath.ps1

```diff
@@ -69,42 +72,38 @@ Get-ToolsLocation
 param(
   [parameter(Mandatory=$true, Position=0)][string] $pathToInstall,
   [parameter(Mandatory=$false, Position=1)][System.EnvironmentVariableTarget] $pathType = [System.EnvironmentVariableTarget]::User,
+  [parameter(Mandatory=$false, Position=2)][switch] $Force = $false,
   [parameter(ValueFromRemainingArguments = $true)][Object[]] $ignoredArguments
 )
-  Write-Debug "Running 'Install-ChocolateyPath' with pathToInstall:`'$pathToInstall`'";
-  $originalPathToInstall = $pathToInstall
+  Set-StrictMode -Version 2

```

Comment:
(The line of code in question was `Set-StrictMode`, not `#Requires`, but the behavior should be the same for both constructs.)
On what systems exactly have you seen that? I'm asking because, as I wrote before, this works as expected on all machines I have access to.
Besides, two widespread Chocolatey extensions (`chocolatey-windowsupdate.extension` and `chocolatey-visualstudio.extension`) use it and there has not been a single issue reported.

Or perhaps you were thinking about executing `powershell.exe -version 2`? This is asking specifically for the 2.0 engine, so naturally it fails if that engine is not available.